embedded-webextension-sdk/step1-hybrid-addon/webextension/content-script.js

```diff
@@ -0,0 +1,4 @@
+browser.runtime.sendMessage({
+  type: "notify-attached-tab",
+  message: window.location.href,
```

Comment:
@wbamberg in a js files it is mostly considered a best practice (which prevent you from forgetting to add a comma when you add new fields to the object), the downside is that the "brain" sometimes start to add the trailing comma to json file automatically, which is annoying, because, on the contrary, makes it an invalid json :-(